tests/examples/README.md

```diff
@@ -0,0 +1,81 @@
+# 5G-MAG Reference Tools: 5GMS Application Server M3 Examples
+
+This directory contains examples of configuration files that can be used with
+the `m3_client_cli.py` tool.
+
+## `Certificates.json`
+
+This file contains a mapping from certificate ID to certificate filename. The
+certificate IDs in this file are used to find the matching certificate file
+(containing a public certificate, private key and any intermediate CA
+certificates) when referenced from a ContentHostingConfiguration file.
+
+The `external/rt-common-shared/5gms/scripts/make_self_signed_certs.py` script
+can be used, passing a ContentHostingConfiguration and this `Certificates.json`
+file as parameters, to create suitable self-signed certificate files for
+testing purposes.
+
+For example:
+```bash
+cd ~/rt-5gms-application-server
+external/rt-common-shared/5gms/scripts/make_self_signed_certs.py tests/examples/ContentHostingConfiguration_Big-Buck-Bunny_pull-ingest_https.json tests/examples/Certificates.json
+```
+
+## `ContentHostingConfiguration_Big-Buck-Bunny_pull-ingest.json`
+
+This file can be used with `m3_client_cli.py` to configuration the
```

Comment:
Would be good to point out that this is an example of the Content Hosting Configuration passed into the 5GMSd AS at reference point M3. As such, it is fully populated, in contrast to the Content Hosting Configuration passed into the 5GMSd AF at reference point M1.